zerver/migrations/0027_realm_subdomain.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.core.exceptions import ObjectDoesNotExist
+
+def set_subdomain_of_default_realm(apps, schema_editor):
+    Realm = apps.get_model('zerver', 'Realm')
+    try:
+        default_realm = Realm.objects.get(domain="zulip.com")
+    except ObjectDoesNotExist:
+        default_realm = None
+    
+    if default_realm is not None:
+        default_realm.subdomain = "zulip"
+        default_realm.save()
```

Comment:
@rishig
> Also, it's possible we should do some sort of migrating in the development environment to set the subdomain for the default "zulip" realm when one migrates to this version...

I made a slight change in zerver/models.py and ran `./manage.py makemigrations`. It was not deleted. Am I missing something?